sdk/textanalytics/azure-ai-textanalytics/README.md

```diff
@@ -28,8 +28,8 @@ and includes six main functions:
 </dependency>
```

Comment:
I can't comment on line 16, but I don't think you addressed the issue's request to have a link for JDK
(From issue #10322 ):
Comment:
Don't have Link for JDK.

Suggestion:
Add following links:
Java Development Kit (JDK) https://docs.microsoft.com/en-us/java/azure/jdk/?view=azure-java-stable

Reply:
I leave a comment about this issue: https://github.com/Azure/azure-sdk-for-java/issues/10322#issuecomment-627777194